packages/core/src/App/Containers/AccountSwitcher/account-switcher.jsx

```diff
@@ -653,6 +654,11 @@ const AccountSwitcher = props => {
                             is_eu={props.is_eu}
                             redirectAccount={() => {
                                 // TODO add the redirect here
+                                const has_logged_in_before = Cookies.get('new_partner_logged_in');
```

Comment:
We usually use localStorage to save details whether our clients saw notifications already or not. I'd suggest that maybe we can use localStorage here as well instead of cookies since clients can reject and clear cookies any time🙏🏻